### DISCUSSION TITLE
请问怎么去掉electerm的水印

### DISCUSSION THREAD

**Discussion by czxin788** [electerm/electerm#2934] [Q&A]
如题，请问怎么去掉electerm的水印

<img width="1378" alt="image" src="https://github.com/electerm/electerm/assets/13861904/7728cbb4-47e1-48c1-b135-bfe246cf281b">

**Comment by zxdong262** [comment]
设置里可以设置背景图片，选择无背景图片即可

**Reply by czxin788** [reply]
明白了，谢谢
